packages/builders/scripts/inter-protocol/manual-price-feed.js

```diff
@@ -102,12 +102,12 @@ export default async function priceAuthorityFromNotifier(
   const offerIt = E(walletAdmin).getOffersNotifier();
   for await (const offers of makeNotifierFromAsyncIterable(offerIt)) {
     // console.log(offers);
-    offers
+    await offers
```

Comment:
This `await` doesn't do anything.  Maybe you mean:
```suggestion
    await Promise.all(offers.filter(...).map(async ({ id }) => { ... }));
```

Reply:
yup!